test/verify/check-packagekit

```diff
@@ -57,6 +57,9 @@ class TestUpdates(MachineCase):
         # no repositories at all, thus no updates
         b.wait_present(".content-header-extra td button")
         b.wait_in_text("#state", "No updates pending")
+        b.wait_present(".content-header-extra td.text-right span")
+        # PK starts from a blank state, thus should force refresh and set the "time since" to 0
+        self.assertEqual(b.text(".content-header-extra td.text-right span"), "Last checked: a few seconds ago")
```

Comment:
It doesn't need to - after calling `RefreshCache()` the code [sets the time since refresh to zero](https://github.com/cockpit-project/cockpit/pull/7055/files#diff-6d5babc8cfb37a2e581d9cd085209d90R577) instead of querying it from PackageKit (this is what happens at page load when a refresh is not needed). So indeed this should assert that the time is exactly zero, and "a few seconds" (momemt.js' translation of "0") is just the tightest value we can check.